utils/terrain/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+
+[package]
+name = "terrain"
```

Comment:
I have a small to moderate preference for using a functional name like `"sorted_vec_map"`, only because `"terrain"` isn't really that good of a name for what we're building.